public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -36,65 +35,71 @@ include ../_util-fns
     .file tsconfig.json
     .file typings.json
 :marked
-  ### Keep the app transpiling and running
-  We want to start the TypeScript compiler, have it watch for changes, and start our server. We'll do this by typing
+  ## Keep the app transpiling and running
+  To start the TypeScript compiler in watch mode and start the server, type the following:
 
 code-example(language="bash").
   npm start
 
 :marked
-  This will keep the application running while we continue to build the Tour of Heroes.
+  This command keeps the application running while you continue to build the Tour of Heroes.
 
 .l-main-section
 :marked
-  ## Displaying Our Heroes
+  ## Displaying heroes
+  To display a list of heroes, you'll add the heroes to the view's template.
+  
   ### Creating heroes
-  Let’s create an array of ten heroes.
+  Create an array of ten heroes.
+//  CF: I don't see a rule in the Angular Writing Guidelines for task titles. Is there a 
+  preference for using gerunds ("Creating heroes") or imperatives ("Create heroes")?
+  Both styles are used in this tutorial.
 
 +makeExample('toh-2/ts/app/app.component.ts', 'hero-array', 'app.component.ts (hero array)')
 
 :marked
-  The `HEROES` array is of type `Hero`, the class defined in part one, 
-  to create an array of heroes.
-  We aspire to fetch this list of heroes from a web service, but let’s take small steps
-  first and display mock heroes.
+  The `HEROES` array is of type `Hero`, the class defined in the previous page.
+  We aspire to fetch the list of heroes from a web service, but you'll take small steps
```

Comment:
Same goes for "we" in this file as in part one. I won't note it each time since you know now.